regexes/client/browsers.yml

```diff
@@ -5,6 +5,11 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Oregano Browser (https://oregan.net/)
+- regex: 'OreganMediaBrowser(?:/(\d+[\.\d]+))?'
+  name: 'Oregano Browser'
+  version: '$1'
+
```

Comment:
if possible, move the rule closer to the oregano browser

Reply:
I've joined both regexes.